src/main/scala/io/iohk/praos/domain/package.scala

```diff
@@ -37,12 +39,17 @@ package object domain {
 
   type SlotNumber = Int
 
+  type Block = Signed[UnsignedBlock]
+  def Block(unsignedBlock: UnsignedBlock, signature: Signature) = Signed[UnsignedBlock](unsignedBlock, signature)
```

Comment:
Maybe we could move this and the `blockHash` function to a separate `object Block` in a separate file?

Reply:
Resolved.